packages/icons/.npmignore

```diff
@@ -21,9 +21,10 @@ node_modules
 src/**/*
 !src/**/*.types.ts
 !src/**/examples/*.tsx
+!src/IconNames.ts
```

Comment:
Why this change?

Reply:
Testing if it would make a difference, it did not, so I'm exploring other solutions.